pipeline_tools/shared/submission/format_map.py

```diff
@@ -19,3 +22,13 @@
     "[.]npz$": "npz",
     "[.]npy$": "npy",
 }
+
+NAMESPACE = uuid.UUID('c6591d1d-27bc-4c94-bd54-1b51f8a2456c')
```

Comment:
Just curious but where does this uuid come from?

Reply:
Good question - this is just an arbitrary value. We need one to use consistently in order for the UUIDs to be deterministic.